plugins/scm.js

```diff
@@ -21,6 +20,20 @@ const token = models.user.base.extract('token').required();
 const type = core.scm.hook.extract('type').required();
 const username = models.user.base.extract('username').required();
 
+const githubStatus = [
+    'PENDING',
```

Comment:
does letter case matter here ?

Reply:
Capital Letter is working as I have verified that. But the `UPDATE_COMMIT_STATUS` object is being used to validate status of builds as well where it is conflicting the purpose. I'm checking that.

Reply:
It is being handled in scm github & gitlab individually.

At other places capital letters are being used.